### PR TITLE
Add missing skipUpdateCheck option to config schema

### PR DESCRIPTION
**Pull request by ryan-case-ml**
Addresses #1010

### CODE REVIEW

**Comment by ryan-case-ml** [comment]
@thafryer i also realized we need to update this in the gha too but im not as sure on how to do that

**Review by thafryer — CHANGES_REQUESTED** [review]
@ryan-case-ml Thanks for opening a PR for this! There are a few adjustments to be made then it should be ready to ship!

File: node-src/lib/getConfiguration.test.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ it('reads configuration successfully', async () => {
       buildScriptName: 'build-script-name',
       outputDir: 'output-dir',
       skip: 'skip',
+      skipUpdateCheck: true,
```
[the remark this answers is not in the document]
Comment (ryan-case-ml) [reply node-src/lib/getConfiguration.test.ts:38]: switch them to false you mean?

File: action-src/main.ts [hunk 2]
```diff
@@ -170,6 +171,7 @@ async function run() {
         projectToken,
         repositorySlug: maybe(repositorySlug),
         skip: maybe(skip),
+        skipUpdateCheck: maybe(skipUpdateCheck),
```
[the remark this answers is not in the document]
Comment (ryan-case-ml) [reply action-src/main.ts]: addressed in [12a4d4a](https://github.com/chromaui/chromatic-cli/pull/1011/commits/12a4d4a56823a9cb71ad7203e43375a22da915ef)

File: node-src/lib/getConfiguration.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (thafryer) [reply node-src/lib/getConfiguration.test.ts:38]: Yes!

**Review by thafryer — APPROVED** [review]
Looks Good!

**Comment by thafryer** [comment]
@ryan-case-ml Thanks so much for your contribution!

**Comment by ghengeveld** [comment]
:rocket: PR was released in [`v11.5.5`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.5.5) :rocket:

**Pull request merged**